### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by corranwebster**
Modify API docs to take into account new modules.

### CODE REVIEW

**Comment by jwiggins** [comment]
Presumably this is the output of some tool... Should an attempt be made to review?

**Comment by corranwebster** [comment]
Yes, this was the output of
```
sphinx-apidoc -e -M -o docs/source/api pyface pyface/wx/* pyface/qt/* pyface/ui/* pyface/dock/* pyface/util/fix_introspect_bug.py pyface/grid/*
```

I'm currently modifying `etstool.py` to autogenerate the docs, including this command.

**Comment by corranwebster** [comment]
A quick look over `etstool.py` and `conf.py` changes for sanity would be appreciated.

File: etstool.py [hunk 1]
```diff
@@ -271,6 +284,45 @@ def update(runtime, toolkit, environment):
...
+    ]
+    click.echo("Installing documentation tools in  '{environment}'".format(
+        **parameters))
+    execute(commands, parameters)
+    click.echo('Done installing documentation tools')
+
+    click.echo("Regenerating API docs in  '{environment}'".format(**parameters))
+    if os.path.exists('docs/source/api/'):
+        rmtree('docs/source/api/')
+    os.makedirs('docs/source/api/')
```
Comment (jwiggins) [inline etstool.py]: You should probably use `os.path.join` for these paths.

File: etstool.py [hunk 2]
```diff
@@ -271,6 +284,45 @@ def update(runtime, toolkit, environment):
...
+    commands = [
+        "edm run -e {environment} -- sphinx-apidoc -e -M -o docs/source/api pyface " + ignore,
+    ]
+    execute(commands, parameters)
+    click.echo('Done regenrating APU docs')
```
Comment (jwiggins) [inline etstool.py]: Typos

File: etstool.py (hunk 1 above)
Comment (corranwebster) [reply etstool.py]: Fixed.

**Pull request merged**